components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -31,37 +31,37 @@ Migrating existing teams...
 Success: Enabled IAM v2.1
 ```
 
-To upgrade without porting existing policies, use: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
+To upgrade without porting existing policies, use the command: `chef-automate iam upgrade-to-v2 --skip-policy-migration`.
 
 ## View Policies
 
-After you've signed in to Chef Automate, select the **Settings** tab in the top navigation bar, then select and locate the `Policies` section in the left navigation.
+After you have signed in to Chef Automate, select the **Settings** tab in the top navigation bar, and then select and locate the `Policies` section in the left navigation.
 
 In this section, you can view all of your v2 policies. If you have upgraded without using the `--skip-policy-migration` flag, you will also see v1 policies.
 
-This includes the following:
+This policy display includes the following:
 
 * New default, *Chef-managed* policies: Administrator, Ingest, Editors, and Viewers.
 * Imported v1 default policies--now called *legacy policies*--in the new v2 policy format and marked with the `[Legacy]` prefix.
-* Imported v1 custom policies that you created; these are marked with the `[Legacy]` prefix and a `(custom)` suffix.
+* Imported v1 custom policies that you created, which are marked with the `[Legacy]` prefix and a `(custom)` suffix.
 
 ![](/images/docs/admin-policies-migrated.png)
 
 ## Policy Conversion
 
-After upgrading we recommend that you reconstitute your v1 policies as v2 policies.
-Once that is done, delete the old legacy v1 policies and you will have an up-to-date system.
+After upgrading, we recommend that you reconstitute your v1 policies as v2 policies.
+Once this policy conversation is done, delete the old legacy v1 policies and you will have an up-to-date system.
```

Comment:
conversation?

Reply:
Clearly, I was talking to myself ;) Fixed. Thanks!